src/app/mainwindow.py

```diff
@@ -15,29 +18,54 @@ class MainWindow(QMainWindow):
     def __init__(self):
         super(MainWindow, self).__init__()
         self.initUI()
+        self.setupMenuBar()
 
     def initUI(self): 
         self.setMinimumSize(500,500)
         self.showMaximized()
-        self.setupMenuBar()
+        #self.setupMenuBar()
 
         # TODO: Create a stack widget and add these views to the stack widget,
         # so that we can navigate between both of these views. 
         # Also, I think QStackWidgets saves the state of each of the widgets
         # that it holds, so that works on our favor. 
+        self.windowStack = QStackedLayout()
+
         self.analysisView = AnalysisView(self)
         self.processingView = ProcessingView(self)
-        self.view = VIEW.ANALYSIS
-        self.setCentralWidget(self.analysisView)
+
+        pic_label = QLabel()
+        home_page = QPixmap("PICK_home.png")
+        pic_label.setPixmap(home_page.scaled(self.width(),self.height(), QtCore.Qt.IgnoreAspectRatio, QtCore.Qt.FastTransformation))
+
+        self.windowStack.addWidget(pic_label)
+        self.windowStack.addWidget(ProjectConfigDialog(self))
```

Comment:
Not what I meant, dialog should not be part of the MainStack. What I meant in the TODO was for something like
`dialog = ProjectConfigDialog(self)

dialog.show()`
Then just remove the .show() that is in the ProjectConfigDialog class